src/main/java/uk/gov/justice/hmpps/prison/service/InmateAlertService.java

```diff
@@ -63,7 +63,6 @@ public InmateAlertService(
         this.maxBatchSize = maxBatchSize;
     }
 
-    @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER", "GLOBAL_SEARCH", "VIEW_PRISONER_DATA"})
```

Comment:
This method is only called as part of offenderDetails - and that endpoint has now been completed (and the overrideRoles added to the Resource file).  This annotation therefore be removed